modules/jlink/tests/features/jlink.feature

```diff
@@ -0,0 +1,15 @@
+@ubi9/openjdk-11
+@ubi9/openjdk-17
+@ubi9/openjdk-21
+Feature: Openshift OpenJDK S2I tests (jlink specific)
+Scenario: Ensure jlinked builder is used to build the containerized application image
+      Given s2i build https://github.com/jboss-container-images/openjdk-test-applications from quarkus-quickstarts/getting-started-3.0.1.Final-nos2i
+       | variable            | value        |
+       | S2I_ENABLE_JLINK    | true         |
+       | QUARKUS_PACKAGE_TYPE| uber-jar     |
+      Then run ls /tmp/jre in container and check its output for bin
+
+Scenario: Ensure S2I_ENABLE_JLINK is not set to true
+      Given s2i build https://github.com/jboss-container-images/openjdk-test-applications from quarkus-quickstarts/getting-started-3.0.1.Final-nos2i
+      Then container log should not contain S2I_ENABLE_JLINK=true
+      And run ls /tmp/jre in container and check its output for ls:
```

Comment:
I guess `ls:` here is a string that ls prints when it doesn't find something, but it's not clear. Could we look for `No such file or directory` instead?